app/backend/resources/languages.json

```diff
@@ -0,0 +1,706 @@
+[
+  {
+    "code": "abk",
+    "name": "Abkhazian"
+  },
+  {
+    "code": "aar",
+    "name": "Afar"
+  },
+  {
+    "code": "afr",
+    "name": "Afrikaans"
+  },
+  {
+    "code": "aka",
+    "name": "Akan"
+  },
+  {
+    "code": "sqi",
+    "name": "Albanian"
+  },
+  {
+    "code": "amh",
+    "name": "Amharic"
+  },
+  {
+    "code": "ara",
+    "name": "Arabic"
+  },
+  {
+    "code": "arg",
+    "name": "Aragonese"
+  },
+  {
+    "code": "hye",
+    "name": "Armenian"
+  },
+  {
+    "code": "asm",
+    "name": "Assamese"
+  },
+  {
+    "code": "ava",
+    "name": "Avaric"
+  },
+  {
+    "code": "ave",
+    "name": "Avestan"
+  },
+  {
+    "code": "aym",
+    "name": "Aymara"
+  },
+  {
+    "code": "aze",
+    "name": "Azerbaijani"
+  },
+  {
+    "code": "bam",
+    "name": "Bambara"
+  },
+  {
+    "code": "bak",
+    "name": "Bashkir"
+  },
+  {
+    "code": "eus",
+    "name": "Basque"
+  },
+  {
+    "code": "bel",
+    "name": "Belarusian"
+  },
+  {
+    "code": "ben",
+    "name": "Bengali"
+  },
+  {
+    "code": "bis",
+    "name": "Bislama"
+  },
+  {
+    "code": "bos",
+    "name": "Bosnian"
+  },
+  {
+    "code": "bre",
+    "name": "Breton"
+  },
+  {
+    "code": "bul",
+    "name": "Bulgarian"
+  },
+  {
+    "code": "mya",
+    "name": "Burmese"
+  },
+  {
+    "code": "cat",
+    "name": "Catalan"
+  },
+  {
+    "code": "cha",
+    "name": "Chamorro"
+  },
+  {
+    "code": "che",
+    "name": "Chechen"
+  },
+  {
+    "code": "nya",
+    "name": "Chichewa"
+  },
+  {
+    "code": "zho",
+    "name": "Chinese"
```

Comment:
Where is this list from? Surprised it has Chinese rather than Cantonese and Mandarin...

Reply:
I think we took the ISO-639-1 codes (2 letter codes) and used those for this subset of 639-3.

Do you think we should remove `zho` and add Cantonese (`yue`) and Mandarin (`cmn`)?

(In 639-3, `zho` is a "macrolanguage" or something, I think that's why it has a general name.)